update-afc.sh

```
@@ -28,6 +28,7 @@ source "${SCRIPT_DIR}/include/install_functions.sh"
 source "${SCRIPT_DIR}/include/uninstall.sh"
 source "${SCRIPT_DIR}/include/update_commands.sh"
 source "${SCRIPT_DIR}/include/update_functions.sh"
+source "${SCRIPT_DOR}/include/emu_templater.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./include/emu_templater.sh was not specified as input (see shellcheck -x).

(SC1091)